client/src/pages/UserPage.js

```diff
@@ -1,5 +1,126 @@
+          <div className="flex items-center justify-center w-32 h-32 rounded-full bg-white">
+            Profile Pic
+          </div>
+          <div className="self-start">
+            <div className="p-4">Name</div>
+            <div className="p-4">Date Joined</div>
+            <div className="p-4">LinkedIn</div>
+            <div className="p-4">Github</div>
+          </div>
+        </div>
+        <div className="flex flex-col w-3/4 rounded-md bg-slate-100">
+          <p className="p-4">User ReadME</p>
+          <div className="h-full p-4 m-8 mt-0 bg-white rounded-md overflow-auto">
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Inventore
+              consectetur temporibus quae aliquam nobis nam accusantium, minima
+              quam iste magnam autem neque laborum nulla esse cupiditate modi
+              impedit sapiente vero?
+            </p>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Inventore
+              consectetur temporibus quae aliquam nobis nam accusantium, minima
+              quam iste magnam autem neque laborum nulla esse cupiditate modi
+              impedit sapiente vero?
+            </p>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Inventore
+              consectetur temporibus quae aliquam nobis nam accusantium, minima
+              quam iste magnam autem neque laborum nulla esse cupiditate modi
+              impedit sapiente vero?
+            </p>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Inventore
+              consectetur temporibus quae aliquam nobis nam accusantium, minima
+              quam iste magnam autem neque laborum nulla esse cupiditate modi
+              impedit sapiente vero?
+            </p>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Inventore
+              consectetur temporibus quae aliquam nobis nam accusantium, minima
+              quam iste magnam autem neque laborum nulla esse cupiditate modi
+              impedit sapiente vero?
+            </p>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Inventore
+              consectetur temporibus quae aliquam nobis nam accusantium, minima
+              quam iste magnam autem neque laborum nulla esse cupiditate modi
+              impedit sapiente vero?
+            </p>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Inventore
+              consectetur temporibus quae aliquam nobis nam accusantium, minima
+              quam iste magnam autem neque laborum nulla esse cupiditate modi
+              impedit sapiente vero?
+            </p>
+          </div>
+        </div>
+      </div>
+      <div className="flex h-1/3 gap-10">
+        <ScrollableList title="Teams" widthString="w-2/3">
```

Comment:
I think you can leave out "String" from "widthString" prop name. "width" is sufficient.